inventory/any.cf

```diff
@@ -520,38 +520,30 @@ bundle agent cfe_autorun_inventory_aws_ec2_metadata
 {
   methods:
     !(disable_inventory_aws|disable_inventory_aws_ec2_metadata)::
-      "cfe_autorun_inventory_aws_ec2_metadata_cache";
+      "cfe_autorun_inventory_aws_ec2_metadata_cache"
+        action => if_elapsed_day;
```

Comment:
Note: Guarding a bundle from being evaluated does not prevent variables within that bundle from being resolved (because of pre-evaluation).

Reply:
oh, right. Thanks!